Sources/MongoSwift/BSON/Document.swift

```diff
@@ -295,11 +335,13 @@ public struct Document: ExpressibleByDictionaryLiteral, ExpressibleByArrayLitera
     }
 
     /// Appends the key/value pairs from the provided `doc` to this `Document`.
+    /// Note: This function does not check for or clean away duplicate keys.
```

Comment:
Is `merge` our own method or part of some protocol? It looks like we only use it in the `Collection.createIndexes()` helper.

Reply:
`merge` seems to be our own function. This note was included because it was something me and Kaitlin realized was not made clear anywhere but may be assumed to be true when its not.